packages/cojson/src/tests/sync.test.ts

```diff
@@ -956,27 +956,6 @@ test.skip("When a peer's incoming/readable stream closes, we remove the peer", a
     */
 });
 
-test("If we start loading a coValue before connecting to a peer that has it, it will load it once we connect", async () => {
```

Comment:
This doesn't make much sense and probably related to some old workaround.

If there are no peers it should be correct to mark to CoValue as unavailable.

Hopefully the changes done on sync should cover the case where the workaround is aimed.